src/components/UsersCard/UsersCard.jsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from 'react-router-dom';
+import './UsersCard.style.css';
+
+const UsersCard = (props) => {
+  const { user } = props;
+  return (
+    <section className='user-list'>
+      <ul className='user-inner-list'>
+        <li>
```

Comment:
why do you have a <ul> and a <li> when only one list item is being rendered here - feels like you're better off just using a non-list element here